internal/devmods.json

```diff
@@ -1,32 +1,32 @@
 {
```

Comment:
Keep JSON/YAML files with an indent of 2. Refer to the [`.editorconfig`](https://github.com/CreatorsTF/Creators.TF-Community-Launcher/blob/master/.editorconfig) file.

Reply:
I auto formatted it with vs code, must not follow the settings from this.
Though 4 space indent is better because this matches tabs better.